src/connector/src/source/base.rs

```diff
@@ -316,8 +316,15 @@ pub fn extract_source_struct(info: &PbStreamSourceInfo) -> Result<SourceStruct>
 /// Stream of [`SourceMessage`].
 pub type BoxSourceStream = BoxStream<'static, crate::error::ConnectorResult<Vec<SourceMessage>>>;
 
-pub trait ChunkSourceStream =
-    Stream<Item = crate::error::ConnectorResult<StreamChunk>> + Send + 'static;
+pub trait ChunkSourceStream:
```

Comment:
To improve IDE experience.

Reply:
You may want to add a comment here to avoid future confusion.